### DISCUSSION TITLE
Provider auth in flutter return null 😩😩

### DISCUSSION THREAD

**Discussion by Ashwaq3** [supabase/supabase#6465] [Questions] [answered]
This is getting me crazy. Tried almost everything it’s been a week now and no success i cant get the current session as it’s null 😭 help me
![image](https://user-images.githubusercontent.com/40840475/163695808-ff69838d-3991-4c91-8fae-94317fa2128e.jpg)

**Comment by dshukertjr** [comment]
Hi @Ashwaq3 

So sorry for the delayed response and sorry to hear that you are struggling to perform Provider login. 

When you actually run the code, does an popup appear to perform apple signin?

**Reply by Ashwaq3** [reply]
> Hi @Ashwaq3
> 
> So sorry for the delayed response and sorry to hear that you are struggling to perform Provider login.
> 
> When you actually run the code, does an popup appear to perform apple signin?

In my Application I have google, apple and Facebook login and all when i press it takes me to their page to sign in and after I sign in it returns me back to the app perfectly. BUT i get null session and user not authenticated 🥲

**Comment by Ashwaq3** [comment]
I tried Supabase-flutter package too and too many things but no luck

**Comment by PcolBP** [comment, marked as answer]
@Ashwaq3 remove hash strategy as written [here](https://docs.flutter.dev/development/ui/navigation/url-strategies), that helps me with null value.

**Reply by Ashwaq3** [reply]
Thanks for the reply.. it’s working I will share the sol. Soon

**Reply by PcolBP** [reply]
@dshukertjr I think that it should be documented somewhere I was struggling for some time really well xD

**Reply by Ashwaq3** [reply]
Agree.. me too I struggled for long time to get it working
